index.html

```diff
@@ -0,0 +1,48 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Apple</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<nav class="top-nav">
+		<ul>
+			<li><i class="fa fa-apple" aria-hidden="true"></i></li>
+			<li>Store</li>
+			<li>Mac</li>
+			<li>iPod</li>
+			<li>iPhone</li>
+			<li>iPad</li>
+			<li>iTunes</li>
+			<li>Support</li>
+			<li class="searchbox-con"><input type="search" name="search" class="searchbox"></li>
+		</ul>
+	</nav>
+
+	<main>
+		<section class="top-header">
+			<p class="top-header-title">What will your verse be?</p>
+			<p class="top-header-subtitle">Everyone has something to share. See how people are using iPad in their unique ways to add to the worlds story.</p>
```

Comment:
All the text inside `top-header` section are also clickable links:
- Each link is an `<a>` tag, with a `<div>` tag inside it.
- The `<div>` tag  inside the `<a>` tag should use an image as background -- [ ] The text on the original page is actually an image